meson.build

```
@@ -130,6 +130,10 @@ thread_dep = dependency('threads') # pthread for tensorflow-lite
 # Protobuf
 protobuf_dep = dependency('protobuf', version: '>= 3.6.1', required: false)
 
+# Flatbuffer compiler
+flatbuf_dep = cc.find_library('flatbuffers')
```

I think building nnstreamer would fail when flatbuf is not present.
```suggestion
flatbuf_dep = cc.find_library('flatbuffers', required: get_option('flatbuf-support'), version: '*if any*')
```

---

Right! I updated required option.